test/Microsoft.Framework.FileSystemGlobbing.Tests/FunctionalTests.cs

```diff
@@ -29,6 +29,17 @@ public void Dispose()
         }
 
         [Fact]
+        public void DuplicatePatterns()
```

Comment:
Why? all the tests should pass on both Windows and *nix. I can add variation for backslash in this test. But a conditional test for Windows only shouldn't exist. We suppose to take both types of slashes.